geocity/apps/accounts/forms.py

```diff
@@ -166,6 +167,14 @@ def __init__(self, *args, **kwargs):
         ),
     )
     captcha = CaptchaField(required=True)
+    policy = forms.BooleanField(
+        required=True,
+        label=mark_safe(
+            _(
+                'J\'ai lu et j\'accepte la <b><a href="https://geocity.ch/legal/confidentialite.pdf" target="_blank">politique de confidentialité</a></b>'
```

Comment:
You need to use the Constance value (like in the footer), not hardcode it

Reply:
+1